src/test/java/io/github/thebusybiscuit/slimefun4/api/events/TestSlimefunRegistryFinalizedEvent.java

```diff
@@ -0,0 +1,45 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
```

Comment:
Static imports are against the code style guidelines ([See here](https://github.com/Slimefun/Slimefun4/blob/master/CONTRIBUTING.md#1-imports))